src/wp-includes/class-wp-theme-json.php

```diff
@@ -488,6 +488,54 @@ public function get_settings() {
 		}
 	}
 
+	/**
+	 * Returns the page templates of the current theme.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @return array
+	 */
+	public function get_custom_templates() {
+		$custom_templates = array();
+		if ( ! isset( $this->theme_json['customTemplates'] ) ) {
```

Comment:
If this element exists, will it always be an array (or iterable)?

If it could be `null`, an empty string, `false` or anything else, the `foreach` will fail and a PHP Warning will be thrown (see in action here https://3v4l.org/rqd0S). May want to add a check here to guard it.

Reply:
Will defer to @ockham again here 🙂

Reply:
Thanks, added in 345ffa8.